gazelle/js/typescript/tsconfig.go

```diff
@@ -140,21 +161,31 @@ func parseTsConfigJSON(cm *TsConfigMap, root, configDir string, tsconfigContent
 	return &config, nil
 }
 
+// Returns the path from the bazel-root to the active tsconfig.json file
+// if the passed path is not absolute.
+// Or an empty string if the path is absolute
+func (c TsConfig) getRelativeExpansionIfLocal(importPath string) string {
+	// Absolute paths must never be expanded but everything else must be relative to the bazel-root
+	// and therefore expanded with the path to the current active tsconfig.json
+	if !path.IsAbs(importPath) {
+		BazelLog.Debugf("Found local path %s in tsconfig.json. Should be expanded with tsconfig dir: %s", importPath, c.ConfigDir)
```

Comment:
Maybe `Tracef`? I think this would be called very frequently?

Reply:
Yes might be better. Just set this to debug for dev reasons (trace was more output and the most was not required for this fix). Will change to trace